tutorial/part3/data_exploitability_pangeo.ipynb

```diff
@@ -2,9 +2,7 @@
  "cells": [
```

Comment:
<p>Maybe would be nice to use the same approach I used in the notebook for warnings just to be more consistent</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-data/pangeo-openeo-BiDS-2023/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>